modules/virt-editing-vm-web.adoc

```diff
@@ -44,7 +44,7 @@ Labels and annotations are editable for both preconfigured Red Hat templates and
 
 . Click the *{object-vm-details}* tab.
 
-. Click the pencil icon to make a field editable.
+. Click any field that has the pencil icon, which indicates that the field is editable. For example, click the current *Boot mode* setting, such as BIOS or UEFI, to open the *Boot mode* window and select a new option from the list.
```

Comment:
LGTM. I think the current UI says "Boot method". I left a comment on a merged UI PR (sorry, can't find the comment right now) suggesting they change the field to "Boot mode". So you might want to check with Aviv Turgeman or Yaacov Zamir about the exact text. OTOH, it's a fairly trivial change and Aviv did say that they would change to "Boot mode" in the future. So probably not worth pursuing.

The only suggestion I have is perhaps making the 2nd sentence more concise since there are only 2 options, addition to the default: "For example, click *Boot mode* to change the default BIOS boot mode to UEFI or UEFI (secure)." Everything else looks fine.

Reply:
Ah, I meant to mention that I checked the UI on a test cluster and it does say "Boot mode" now! And... I wrote the second sentence in that way because I was way overthinking your comment about preventing future maintenance. 🤦